mllib/src/main/scala/org/apache/spark/mllib/feature/Word2Vec.scala

```diff
@@ -315,13 +315,15 @@ class Word2Vec extends Serializable with Logging {
     val syn0Global =
       Array.fill[Float](vocabSize * vectorSize)((initRandom.nextFloat() - 0.5f) / vectorSize)
     val syn1Global = new Array[Float](vocabSize * vectorSize)
+    val bcSyn0Global = sc.broadcast(syn0Global)
```

Comment:
I got where it went wrong.
I tested on the cluster with a different edition
```
val value = bc.value
      val result = rdd.map(i => value(2)).collect().mkString(", ")
```

Anyway, you are correct. Thanks